src/main/java/com/evilcorp/bloom/model/Product.java

```diff
@@ -8,16 +8,97 @@ public class Product {
   @Id
   private String id;
 
-  private String name;
+  private String productName;
   private String description;
-  private String category;
-  private String brand;
+  private Integer category;
+  private Integer brand;
   private String imgUrl;
 
   private Double price;
   private Double cost;
   private Integer stockQty;
 
-  private String createdAt;
-  private String UpdatedAt;
+  public Product(String productName, String description, Integer category, Integer brand,
```

Comment:
It is fine, the list of arguments is pretty big. 
It would be interesting to consider the builder design pattern to simplify its use

Reply:
you can make it part of the issue that you created for the builder

Reply:
ok